setup.py

```diff
@@ -4,7 +4,7 @@
 
 # packages required for local development and testing
 development = [
-    "black==22.3.0",
+    "black==22.4.0",
```

Comment:
we need to update the bumpversion.cfg file to only search for `version="{current_version}",`

Reply:
Yes